tests/kernel/arm_zero_latency_irqs/README.txt

```diff
@@ -0,0 +1,45 @@
+Title: Test to verify the behavior of CONFIG_ZERO_LATENCY_IRQS at runtime (ARM Only)
+
+Description:
+
+This test verifies the behavior of CONFIG_ZERO_LATENCY_IRQS at runtime.
+In particular, it tests that IRQs configured with the IRQ_ZERO_LATENCY
+flag are assigned the highest priority in the system (and, therefore,
+cannot be masked-out by irq_lock()).
+Only for ARMv7-M and ARMv8-M Mainline targets.
+
+---------------------------------------------------------------------------
+
+Building and Running Project:
+
+This project outputs to the console.  It can be built and executed on QEMU as
+follows:
+
+    make run
+
+---------------------------------------------------------------------------
+
+Troubleshooting:
+
+Problems caused by out-dated project information can be addressed by
+issuing one of the following commands then rebuilding the project:
+
+    make clean          # discard results of previous builds
```

Comment:
Let's use `ninja/make clean` and `ninja/make pristine`

Reply:
Will update this, tnem

Reply:
Fixed.